### PR TITLE
[SIG 4359] Show icon and label below map preview on summary page

### PR DESCRIPTION
**Pull request by kiruys**

### CODE REVIEW

File: src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx [hunk 1]
```diff
@@ -20,13 +20,21 @@ import type {
...
 const mapWidth = 640
 const mapHeight = 300
 const mapZoom = 12
 
-const Address = styled.address`
-  margin-bottom: ${themeSpacing(4)};
-  font-style: normal;
+const LocationDescription = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (janjaap) [inline src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx:30]: No need to define `row`; this is the default value

File: src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.test.tsx [hunk 2]
```diff
@@ -159,4 +160,52 @@ describe('signals/incident/components/IncidentPreview/components/MapPreview', ()
...
     expect(screen.queryByTestId('mapStaticMarker')).not.toBeInTheDocument()
   })
+
+  describe('Location description', () => {
+    it('When an object is selected', () => {
+      render(withAppContext(<MapPreview {...props} />))
+      expect(screen.queryByTestId('locationDescription')).toBeInTheDocument()
+      expect(screen.queryByTestId('typeIcon')).toBeInTheDocument()
+      expect(screen.queryByTestId('selectionLabel')).toBeInTheDocument()
+      expect(screen.queryByTestId('mapAddress')).toBeInTheDocument()
```
Comment (boris-arkenaar) [inline src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.test.tsx:170]: Please try to make use of other queries over `...ByTestId`. See https://testing-library.com/docs/queries/about#priority.

Comment (kiruys) [reply src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.test.tsx:170]: sure

File: src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx [hunk 3]
```diff
@@ -85,6 +89,25 @@ const MapPreview: FC<MapPreviewProps> = ({ incident, value, featureTypes }) => {
...
+        {iconSrc && (
+          <StyledImg
+            data-testid="typeIcon"
+            alt=""
+            src={iconSrc}
+            height={ICON_SIZE}
+            width={ICON_SIZE}
+          />
+        )}
+        <div>
```
[the remark this answers is not in the document]
Comment (kiruys) [reply src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx:102]: if the div is replaced by a fragment, the description and the address appear on the same line.

File: src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx (hunk 1 above)
Comment (kiruys) [reply src/signals/incident/components/IncidentPreview/components/MapPreview/MapPreview.tsx:30]: ✅

**Comment by kiruys** [comment]
I have replaced the MapPreview component by the Summary component. The MapPreview component is removed from the codebase.

File: src/signals/incident/components/IncidentPreview/IncidentPreview.test.tsx [hunk 4]
```diff
@@ -8,6 +8,8 @@ import { withAppContext } from 'test/utils'
 import { formatAddress } from 'shared/services/format-address'
 import { mock } from 'types/incident'
 
+import Summary from 'signals/incident/components/form/MapSelectors/Asset/Summary'
```
[the remark this answers is not in the document]
Comment (kiruys) [reply src/signals/incident/components/IncidentPreview/IncidentPreview.test.tsx]: okay

File: src/signals/incident/components/IncidentPreview/IncidentPreview.test.tsx [hunk 5]
```diff
@@ -215,7 +217,7 @@ describe('<IncidentPreview />', () => {
       expect(queryByText(incident.datetime.label)).toBeInTheDocument()
       expect(
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion,@typescript-eslint/no-non-null-asserted-optional-chain
-        queryByText(formatAddress(incident.location?.address!))
+        queryByText(formatAddress(summaryProps.address!))
```
[the remark this answers is not in the document]
Comment (kiruys) [reply src/signals/incident/components/IncidentPreview/IncidentPreview.test.tsx]: ✔️

**Review by janjaap — APPROVED** [review]

**Pull request merged**